diff --git a/survey-creator-core.js b/survey-creator-core.js
index 7229dce2c714650d8fb5447910282221735fe9ba..59a99f3f07a3211b42910426053701e8e662af67 100644
--- a/survey-creator-core.js
+++ b/survey-creator-core.js
@@ -3110,7 +3110,7 @@ var SurveySimulatorModel = /** @class */ (function (_super) {
     return SurveySimulatorModel;
 }(survey_core__WEBPACK_IMPORTED_MODULE_1__["Base"]));
 
-var DEFAULT_MONITOR_DPI = (typeof window !== "undefined" ? window.devicePixelRatio : 1) * 96;
+var DEFAULT_MONITOR_DPI = 96;
 var simulatorDevices = {
     desktop: {
         deviceType: "desktop",
